### PR TITLE
Search use portal config

### PR DESCRIPTION
**Pull request by SaschaBrunnerCH**

### CODE REVIEW

**Review by tomwayson — CHANGES_REQUESTED** [review]

File: app.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* import esri styles */
 /* TODO: these should be lazy loaed by esri-loader's loadModules() */
-@import url('https://js.arcgis.com/4.10/esri/css/main.css');
+@import url('https://js.arcgis.com/4.11/esri/css/main.css');
```
[the remark this answers is not in the document]
Comment (SaschaBrunnerCH) [reply app.css]: @tomwayson: change css to lazy-load

File: utils/map.js [hunk 2]
```diff
@@ -18,10 +18,11 @@ export function coordsToExtent(coords) {
...
 export function newMap(element, mapOptions) {
   // lazy-load the map modules
-  return loadModules(['esri/Map', 'esri/views/MapView', 'esri/Graphic'], {
-    // TODO: also lazy-load the CSS
-    // css: 'https://js.arcgis.com/4.11/esri/css/main.css'
-  }).then(([Map, MapView, Graphic]) => {
+  return loadModules(
+    ['esri/Map', 'esri/views/MapView', 'esri/Graphic'],
+    // NOTE: keep this current w/ the latest version of the JSAPI
+    { css: 'https://js.arcgis.com/4.11/esri/css/main.css' }
```
Comment (tomwayson) [inline utils/map.js]: https://github.com/tomwayson/create-arcgis-app/pull/35#discussion_r280074678

**Review by tomwayson — CHANGES_REQUESTED** [review]
Thanks again! Just update to `css: true` and I'll merge it.

File: utils/map.js (hunk 2 above)
Comment (SaschaBrunnerCH) [reply utils/map.js]: Cool feature, good to know. Updated.

**Review by tomwayson — APPROVED** [review]

**Pull request merged**